src/components/cart.js

```diff
@@ -324,7 +377,13 @@ export default class Cart extends Component {
   updateItem(id, quantity) {
     this._userEvent('updateItemQuantity');
     const lineItem = {id, quantity};
-    this.updateCacheItem(id, quantity);
+    const lineItemEl = this.view.document.getElementById(id);
+    if (lineItemEl) {
+      const quantityEl = lineItemEl.getElementsByClassName(this.classes.lineItem.quantity)[0];
+      if (quantityEl) {
+        addClassToElement('is-loading', quantityEl);
```

Comment:
Just a sanity check but after we get the new checkout model back, it'll trigger re-render and that would remove the loading state right?